web/modules/custom/ai_screening_project_track/src/Helper/ProjectTrackHelper.php

```diff
@@ -320,4 +321,31 @@ private function computeWebformSubmissionReportValues(ProjectTrackInterface $tra
     return $trackConfig;
   }
 
+  /**
+   * Get the quadrant that contains the evaluation.
```

Comment:
Are we using the quadrant numbers from https://da.wikipedia.org/wiki/Kvadrant (https://en.wikipedia.org/wiki/Quadrant_(plane_geometry))? If so, consider adding a reference to https://da.wikipedia.org/wiki/Kvadrant (or similar) to help us remember this in the future.